app/views/design-system/rsd-design-system/design-patterns/page-not-found/index.html

```diff
@@ -0,0 +1,127 @@
+        &lt;/p&gt;
+    &lt;/div&gt;
+  &lt;/div&gt;
+{% endblock %}
+</code>' %}
+
+{% block beforeContent %}{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    {{ sample.code('/design-system/rsd-design-system/design-patterns/page-not-found/example.html', 'Page not found', 'page-not-found',
+    htmlCode, nunjucksCode) }}
+    <br>
+
+    <h2 id="when-to-use-this-pattern">When to use this pattern</h2>
+    <p>Use a page not found if someone is trying to view a page that does not exist. This happens if someone:</p>
+    <ul>
+      <li>selects a link or button that takes them to a page that does not exist</li>
+      <li>types or copies a web address for a page that does not exist</li>
+      <li>types or copies a web address incorrectly</li>
+    </ul>
+    <p>Test all links and buttons to make sure they work. Remember to <a
+        href="https://www.gov.uk/guidance/government-design-principles#do-the-hard-work-to-make-it-simple">do the hard work
+        to make it simple</a>.</p>
+    <p>Make sure any web addresses in your service, letters, forms and on GOV.UK are for pages that exist or redirect to
+      pages that exist.</p>
+      
+    <h2 id="how-it-works">How it works</h2>
+    <p>The page should have:</p>
+    <ul>
+      <li>‘Page not found – System name’ as the page title. For example, Page not found - Manage free school projects</li>
+      <li>‘Page not found’ as the H1</li>
+      <li>'email Service Support for help with using this system' as the link text</li>
+      <li>'System name: support query' as the pre-filled email subject. For example, Manage free school projects: support query</li>
+    </ul>
+    <p>The content should be clear and concise, not blame the user.</p>
+    <div class="app-wcag-22" id="wcag-consistent-content-page-not-found" role="note">
+      <strong class="govuk-tag" style="margin-bottom: 10px">
+        WCAG 2.2
+      </strong>
+    
+      <p>You must always write contact information in a clear and consistent way across ‘Page not found’ and similar service
+        error pages. This relates to WCAG 2.2 success criterion <a
+          href="https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html">3.2.6 Consistent Help</a>.</p>
+    </div>
+    
+    <p>Do not use:</p>
+    <ul>
+      <li>breadcrumbs</li>
+      <li>technical jargon like 404 or bad request</li>
+      <li>informal or humorous words like oops</li>
+      <li>red text to warn people</li>
+    </ul>
+    
+
+
+
+    <h2>Research on this pattern</h2>
+    <p>
+      We have not done research on this pattern, but it is being used in <a
+      href="https://design-histories.education.gov.uk/manage-free-school-projects">Manage free school projects</a> to meet this need from our <a
+        href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user
```

Comment:
Internal link again

Reply:
In the updated version, this part is now:

`
We have not done research on this pattern, but it is being used in <a href="https://design-histories.education.gov.uk/manage-free-school-projects">Manage free school projects</a> to meet this need from our <a href="https://educationgovuk.sharepoint.com/sites/ServiceDeliveryDirectorate/Lists/RSD%20crossprogramme%20user%20needs/AllItems.aspx?env=WebViewList">user needs database (opens in DfE Microsoft List)</a>
`